lifechanger/src/styles.css

```diff
@@ -1,4 +1,9 @@
 /* You can add global styles to this file, and also import other style files */
+/* -------------- `14-18.4` Change the colour names, 
```

Comment:
I think this type of comment is actually fine, we can leave it

Reply:
Thank you, Mentor.
I added it here to not forget about further enhancement and implementation.